scripts/serverless.js

```diff
@@ -609,6 +611,10 @@ processSpanPromise = (async () => {
         if (isHelpRequest) return;
         if (!_.get(variablesMeta, 'size')) return;
 
+        // We do not need to attempt resolution of further variables for login command as the only variable from configuration that we
+        // potentially rely on is `console`
+        if (command === 'login') return;
```

Comment:
Actually a better fix for that, would be to configure `login` as not Serverless instance integrated command. Similarly as now  e.g.`plugin install` or `doctor` (with implementation in  https://github.com/serverless/serverless/tree/main/commands)

But if that's too much hassle, I'd move this conditional higher, as with above we still attempt to resolve most variable sources and crash on eventual errors, when it seems totally fine to not resolve any srouces at all (all we need to know it's wether `configuration.console` is truthy).

So instead I'd put this conditional in those two code points:

- https://github.com/serverless/serverless/blob/0a0f6c6102b3cb8921ef7448d5b3a8f1294b6802/scripts/serverless.js#L238
- https://github.com/serverless/serverless/blob/0a0f6c6102b3cb8921ef7448d5b3a8f1294b6802/scripts/serverless.js#L576


And add a `TODO` note, that it should be removed once we remove dashboard/login URL triage (as then this command should again be configured as service independent)

Reply:
I agree that it would be better to make `login` standalone, but it's a bit of an hassle at this point as the `login` command logic is a part of `dashboard-plugin` at the moment. I think once we merge that repo with Framework, it will be good to reconfigure that command. I've followed your suggestion with the checks.